recipes/tip/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "tip" %}
+{% set version = "1.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/IM-USAF/tip/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 4ba6f6a9e652fa48a99534997aaaa0921a94af0ef2d65ce909b6825c06a2b259
+
+build:
+  skip: true  # [win and vc<14 or osx]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - make 4.*  # [unix]
+    - ninja 1.*
+    - cmake 3.*
+    - pkgconfig 1.*
+
+  host:
+    - libpcap 1.*
+    - arrow-cpp 6.*
+    - spdlog 1.*
+    - yaml-cpp 0.*
+    - gmock 1.*
+    - gtest 1.*
```

Comment:
It also looks like static linking for gmock/gtest on Windows only, but it's Windows so I'm not sure you can do anything about it.

Reply:
The conda-forge gtest and gmock libs are now dynamically linked. (Note is was a nightmare because documentation is non-existent.)

Reply:
Thanks!